src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -1,33 +0,0 @@
-{
-  "dependencies": {
```

Comment:
In CoreFX the project.json used for test runtime seems to be this https://github.com/dotnet/corefx/blob/master/src/Common/test-runtime/project.json while CoreCLR does not have any reference to this. I have tested this change with either products successfully, but it would make sense to take a second opinion.